compile.js

```diff
@@ -13,7 +13,9 @@
       flickrOptions = env.get("FLICKR"),
       APIBuilder = require("./src/flickr-api-object.js"),
       Utils = require("./src/utils.js"),
-      filename = "flickrapi";
+      filename = "flickrapi",
+      fs = require("fs"),
+      wrapper = fs.readFileSync("./src/wrapper.js", 'utf8').split(/\/\/\@CODE\n/);
```

Comment:
it's probably easier to not use a regex here, to keep things legible, so just splitting on `.split('//@CODE\n')';`